generators/app/index.js

```diff
@@ -57,6 +59,12 @@ module.exports = class extends Generator {
       },
       {
         type: 'input',
+        name: 'githubOrgName',
+        message: 'What is the name of your GitHub organization (Leave it blank for your GitHub user)?',
+        default: '',
```

Comment:
I would prefer this to be after we prompt them for the github username so that we could set the default to their username, but since this will likely change soon with #17 let's just keep it like this for now.